pkgs/apps/turbomole/default.nix

```diff
@@ -107,15 +108,19 @@ in stdenv.mkDerivation rec {
       glibc
     ];
   in ''
-    # Patch elf dynamic library paths.
+    # Patch elf dynamic library paths and interpreters
     for dir in $TURBODIR/libso $TURBODIR/bin/*; do
-      files=$(find $dir -type f -executable -maxdepth 1 ! -name rimp2 ! -name ccsdf12)
+      files=$(find $dir -type f -executable -maxdepth 1)
       for f in $files; do
-        # echo "Patching $f now!"
-        patchelf \
-          --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-          --set-rpath "$libSearchPath" \
```

Comment:
The `$libSearchPath` variable is empty here (needs to be `${libSearchPath}`)? patchelf has a segfault bug, when an empty $rpath is used (https://github.com/NixOS/patchelf/pull/319). Could that be it?

Reply:
Ouh, you are completely right. This was the issue and is the better solution. Fixed it and works like this.